psat_server_web/atlas/atlas/models.py

```diff
@@ -1048,7 +1082,7 @@ class TcsGravityAlerts(models.Model):
     creator = models.CharField(max_length=30, blank=True, null=True)
     group = models.CharField(max_length=100, blank=True, null=True)
     pipeline = models.CharField(max_length=100, blank=True, null=True)
-    map_iteration = models.CharField(max_length=100, blank=True, null=True)
+    map_iteration = models.CharField(max_length=100, unique=True, blank=True, null=True)
```

Comment:
This is always requested by Django, but I don't believe it needs to be actually done. Ken to find out why!